docs/root/configuration/network_filters/redis_proxy_filter.rst

```diff
@@ -56,6 +55,8 @@ The Redis filter will gather statistics for commands in the
   :widths: 1, 1, 2
 
   total, Counter, Number of commands
+  success, Counter, Number of commands that were successful
+  error, Counter, Number of commands that have resulted in errors
```

Comment:
nit: may want to clarify "Number of commands that returned a partial or complete error response"